.github/workflows/pr-build.yaml

```diff
@@ -272,3 +289,7 @@ jobs:
           name: cluster_dump_${{github.sha}}
           path: "output"
           retention-days: 5 # Default 90 days
+      - name: Delete Capact CLI artifact
```

Comment:
🤔 Hmm, maybe the artifact deletion is not necessary after all?
I think this is a nice feature to have built CLI for a given PR, similarly how we have the Docker images built. But it would be great to build binary also for Mac, which will increase the build time a little bit, but it would be great for PR review

So two options:
1.  Build also CLI binary for mac, do not delete artifacts, keep the `retention-days` low (e.g. 3 days? 🤔)
2. Do it exactly as it is right now

I'm fine with both approaches, I just wanted to highlight that it could be converted to nice feature for us.

Reply:
For now, I will leave it as it is. Building CLI locally takes around 7sec. Finding the artifact on GH, downloading it, adding perms will take much longer, so personally I would never use it. Additionally, it's not compressed, so it's ~50mb.